src/mjolnir/graphvalidator.cc

```diff
@@ -622,6 +622,9 @@ void GraphValidator::Validate(const boost::property_tree::ptree& pt) {
     LOG_WARN((boost::format("Possible duplicates at level: %1% = %2%") % std::to_string(level) %
               duplicates[level])
                  .str());
+    if (densities[level].empty()) {
+      continue;
+    }
```

Comment:
`float average_density = sum / densities[level].size();`
UB if  `densities[level]` is empty

Reply:
@merkispavel I think we need to get these merged to master https://github.com/SvetlanaBayarovich/valhalla/pull/1  cc @SvetlanaBayarovich

Reply:
> @merkispavel I think we need to get these merged to master [SvetlanaBayarovich#1](https://github.com/SvetlanaBayarovich/valhalla/pull/1) cc @SvetlanaBayarovich

makes sense, I'll drop it from my PR then